appinventor/components/src/com/google/appinventor/components/runtime/ListPicker.java

```diff
@@ -61,6 +63,13 @@
                                 // if left blank, the App Name is used instead
   private boolean resumedFromListFlag  = false; //flag so onResume knows if the resume was triggered by closing the listpicker activity
 
+  private YailList deletedItems = new YailList();
```

Comment:
@kkashi01 What's this deletedItems thing? I don't find a reference to it anywhere. I have removed it from the copy I am testing.